pom.xml

```diff
@@ -44,19 +44,43 @@
             </roles>
             <timezone>+4</timezone>
         </developer>
+        <developer>

```

Comment:
After this change you will be listed as plugin maintainer on the plugin's Wiki page. Is it what you want?